newIDE/app/src/MainFrame/index.js

```diff
@@ -1723,13 +1724,31 @@ const MainFrame = (props: Props) => {
   const launchNewPreview = React.useCallback(
     async options => {
       const numberOfWindows = options ? options.numberOfWindows : 1;
-      launchPreview({ networkPreview: false, numberOfWindows });
+      await launchPreview({ networkPreview: false, numberOfWindows });
     },
     [launchPreview]
   );
 
+  const launchPreviewWithScreenshot = React.useCallback(
```

Comment:
Consider not adding yet-another-way-to-launch-preview and instead:
- Make the logic related to the timing inside the existing function.
- delayTimeInSeconds can also be in the function
- forcing a screenshot, bypassing the check for timing, is "just" a "forceScreenshot: true". But this is only used for quick customization. For normal previews, the buttons are unchanged.